src/toil_scripts/lib/programs.py

```diff
@@ -45,6 +47,7 @@ def docker_call(tool,
                       the environment variable.
     """
     from toil_scripts.lib.urls import download_url
+    import sys
```

Comment:
Move imports to top of file, make imports comply with PEP8/repository conventions.